bootstrap.bash

```diff
@@ -42,7 +42,7 @@ if [ "$(printf "%s\n1.20\n" "$go_version" | sort -t '.' -k 1,1 -k 2,2 -g | head
     echo 'WARNING: Go >=1.20 is not fully supported by Bob/Blueprint and requires a pre-installed Go `std` package on the system. Attempting workaround. You may disable this workaround by setting `GO120_NO_STD_INSTALL=1`'
 
     ret=0
-    (GODEBUG=installgoroot=all go install std) || ret=$?
+    (GODEBUG=installgoroot=all CC="gcc" go install std) || ret=$?
```

Comment:
I don't think we can hardcode the compiler here, instead I would recommend that you pre-install the go std package using this command in your build envrioment and run with `GO120_NO_STD_INSTALL=1` to skip the check if needed.

Reply:
thanks for your suggestion.